src/App.tsx

```diff
@@ -1,14 +1,80 @@
-/* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
-import { TodoList } from './components/TodoList';
+import { TodoList } from './components/TodoList/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
+
+enum SelectedStatus {
+  All = 'all',
+  Active = 'active',
+  Completed = 'completed',
+}
+
+function isIncludesQuery(todoTitle: string, query: string) {
+  return todoTitle.toLowerCase().includes(query.toLowerCase());
+}
+
 export const App: React.FC = () => {
+  const [initialTodos, setInitialTodos] = useState<Todo[]>([]);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [query, setQuery] = useState<string>('');
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+  const [selectedTodoStatus, setSelectedTodoStatus] = useState<string>('all');
```

Comment:
You already have an enum for `all` value
```suggestion
  const [selectedTodoStatus, setSelectedTodoStatus] = useState<string>(SelectedStatus.All);
```